Update Node List Info (#1418)
mongodb/docs-realm: source/sdk/node/data-types/collections.txt

Lists
-----

> A :js-sdk:`List <Realm.List.html>` represents a collection of objects of a single 

Reviewer: ```suggestion
A :js-sdk:`List <Realm.List.html>` represents a collection of values of a single 
```

nit: I think this works better given the thrust of the ticket is that Lists hold more than just objects, at the risk of being repetitive with the next paragraph

type. Lists are declared as a property within an 
:ref:`object model <node-object-schemas>`, and are not associated with a query.

Lists behave like normal JavaScript arrays, except that they 

> can only store values of a single type, which you indicate with the 

Reviewer: ```suggestion
can only store values of a single type, which you indicate by setting the
```
related to my other suggestions in this paragraph

> :js-sdk:`type <Realm.List.html#type>` property set to one of the 
> valid :js-sdk:`property types <Realm.html#~PropertyType>` or another Realm 

Reviewer: ```suggestion
:js-sdk:`type <Realm.List.html#type>` property to one of the 
valid :js-sdk:`property types <Realm.html#~PropertyType>` or the name of another Realm 
```
